src/main/plugins/org.dita.base/build_init.xml

```diff
@@ -45,6 +45,22 @@ See the accompanying LICENSE file for applicable license.
     </sequential>
   </macrodef>
 
+  <macrodef name="preprocess-skip-init">
+    <attribute name="name"/>
+    <attribute name="step"/>
+    <sequential>
+      <fail if:set="@{name}.skip">Internal property @{name}.skip may not be set directly.</fail>
```

Comment:
We may want to change this into an error, not sure. And we could give this error an error code.